src/tests.c

```diff
@@ -5962,142 +5916,99 @@ static void run_ec_pubkey_parse_test(void) {
         0xB8, 0x00
     };
     unsigned char sout[65];
-    unsigned char shortkey[2];
+    unsigned char shortkey[2] = { 0 };
```

Comment:
nit:

```suggestion
    unsigned char shortkey[2] = { 0x03, 0x05 };
```

0x0500000000000000000000000000000000000000000000000000000000000000 is a valid x-coord with odd y-coord

Reply:
I'm not sure how this would improves the tests.

Reply:
Yeah sorry, I didn't think it through entirely.